ctype.md

```diff
@@ -2,50 +2,48 @@
 
 Before the <span class="label-role claimer">claimer</span> can make a claim about themselves, they first need a claim type (CTYPE for short).
 
-A claim type (CTYPE for short) is a KILT-specific term, but the concept is simple.  
+A claim type (CTYPE for short) is a KILT-specific term, but the concept is simple:  
 A CTYPE defines the structure of a claim. You can think of it as the data model for your claim.
 
 For example, a very basic CTYPE for a driver's license could look like this:
 
 ```json
 {
   "schema": {
-    "$id": "KILT_DRIVERS_LICENSE",
+    "$id": "SimpleLicense",
     "$schema": "http://kilt-protocol.org/draft-01/ctype#",
     "properties": {
       "name": {
         "type": "string"
       },
-      "canDrive": {
-        "type": "boolean"
+      "age": {
+        "type": "integer"
       }
     },
     "type": "object"
   },
-  "owner": "8UPfLqrqbKoKyi61YErwUrWEa1PWxikEojV7PCnLJgxrWd6W",
-  "hash": "0x081955a2b7990554f6193a9e770ea625c68d2bfc5a1ff996e6e28d2a620fae16"
+  "owner": "5EkPdXVsoJ8Vm8UTrs7c89ghJpWVZHGN7RGiQFSi6VC6RiX9",
+  "hash": "0x16fa158d730aa83f6339497fafa59d895c89c3eae24dc323532f71e235824152"
 }
 ```
 
-> 💡 JSON Schema  
-> CTYPES are based on JSON Schema, a standard used to annotate and validate JSON documents. We don't need to dive into it in this tutorial, for now we can think of CTYPE as JSON objects.
+> 💡 CTYPES are based on JSON Schema, a standard used to annotate and validate JSON documents. We don't need to dive into it in this tutorial, for now we can think of CTYPE as JSON objects.
```

Comment:
Suggestion: 
 `CTYPES` should be `CTYPE`s.

Reply:
👍right, fixed!